committee-steering/governance/working-group-formation-process.md

```diff
@@ -0,0 +1,87 @@
+# Kubernetes Working Group Formation Process - Proposal
+
+Draft 1.0 // Jaice Singer DuMars // June, 2018
+
+## Process Overview and Motivations
+As opposed to “Birds of a Feather” or discussion groups, Working Groups provide a more formal avenue for disparate
+groups to intellectually collaborate around a common problem, craft a balanced position, and disband. Because they
+represent the interests of multiple special interest groups, they are a vehicle for consensus building, not
+construction. By design, the code under discussion resides in either multiple parts of the project, or is in the
```

Comment:
This sentence seems to require that there be code under discussion, but it's not clear that's always the case. In any event, I think this part should be broken out more crisply.

Something like:

"If the output of the working group is code, that code will be housed in an appropriate repository as described in the [repositories document](https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md). The merging of this code into the repository will be governed by the standard policies regarding submitting code to that repository."

Reply:
PTAL